app/database/users/functors.go

```diff
@@ -0,0 +1,15 @@
+package users
+
+import (
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+)
+
+// OnlyDescendantsOf filters users by joining `groups_ancestors`
+// on idGroupAncestor=groupID & idGroupChild=users.idGroupSelf
+func OnlyDescendantsOf(groupID int64) database.Functor {
```

Comment:
this means `so that users are descendants of ...`. Let's get rid of `Only`?

Reply:
probably it would be better to have `SoThat(items.AreVisibleBy(user))` instead of `SoThat(items.OnlyVisibleBy(user))` and `SoThat(users.AreDescendantsOf(groupID))` instead of `SoThat(users.OnlyDescendantsOf(groupID))`